styles/google-calendar/catppuccin.user.less

```
@@ -0,0 +1,414 @@
+/* ==UserStyle==
+@name Google Calendar Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/google-calendar
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/google-calendar
+@version 2025.03.20
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/google-calendar/catppuccin.user.less
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Agoogle-calendar
+@description Soothing pastel theme for Google Calendar
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain("calendar.google.com") {
+    .DO879e:not(.PC2yOb).CcsDpe {
+        #catppuccin(@darkFlavor);
+    }
+
+    .DO879e:not(.PC2yOb):not(.CcsDpe) {
+        #catppuccin(@lightFlavor);
+    }

```

```suggestion
    body:not(.CcsDpe) {
        #catppuccin(@lightFlavor);
    }

    body.CcsDpe {
        #catppuccin(@darkFlavor);
    }
```
Couldn't something like this work?

---

I think more than just the body is targeted which is why the class name is there.

---

In my testing the selector doesn't get enough specificity to override the existing dark and light themes: I'll leave it as is for now but will add comments to make it more clear what the verbose class lists do.